### PR TITLE
Add support for ddd-react-redux

### PR DESCRIPTION
**Pull request by carlosvillu**
@SUI-Components/developers Review

### CODE REVIEW

File: src/components/demo/EventsButtons.js [hunk 1]
```diff
@@ -1,45 +1,38 @@
...
-    ...base,
-    ...params
-  })
+  domain.get(useCase).execute({ ...params })
 }
 
-const EventsButtons = ({events, domain}) => {
-  if (!domain) { return null }
+const EventsButtons = ({events, store, domain}) => {
+  if (!store) { return null }
```
Comment (naxhh) [inline src/components/demo/EventsButtons.js]: I get a bit lost here.

Why is the store a requirement for the buttons?

For example, there will be components not using the store but using events.

Comment (carlosvillu) [inline src/components/demo/EventsButtons.js]: Sure. I am working on that. And now we are passing the store or the domain or nothing. And checking for both.

File: src/components/demo/EventsButtons.js [hunk 2]
```diff
@@ -1,45 +1,48 @@
...
-  const base = reqFixturesDemo(`./demo/fixtures/${useCase}.js`).default
-  domain.get(useCase).execute({
-    ...base,
-    ...params
+  domain.get(useCase).execute({ ...params })
+}
+
+const dispatchEvent = ({store, domain, useCase, params}) => {
+  domain.get(useCase).execute(params).then(result => {
+    store.dispatch({
```
Comment (naxhh) [inline src/components/demo/EventsButtons.js]: `store && store.dispatch` ?

File: src/components/demo/EventsButtons.js [hunk 3]
```diff
@@ -1,45 +1,48 @@
...
 
-const reqFixturesDemo =
-  require.context(`${__BASE_DIR__}`, true, /\/demo\/fixtures\/.*\.js/)
-
 const executeUseCase = ({domain, useCase, params}) => {
-  const base = reqFixturesDemo(`./demo/fixtures/${useCase}.js`).default
-  domain.get(useCase).execute({
-    ...base,
-    ...params
+  domain.get(useCase).execute({ ...params })
```
Comment (naxhh) [inline src/components/demo/EventsButtons.js]: why params is deconstructed and constructed again here?

File: src/components/demo/EventsButtons.js [hunk 4]
```diff
@@ -1,45 +1,48 @@
...
+}
+
+const dispatchEvent = ({store, domain, useCase, params}) => {
+  domain.get(useCase).execute(params).then(result => {
```
Comment (naxhh) [inline src/components/demo/EventsButtons.js]: you can reuse `executeUseCase` here.

```
executeUseCase({domain, useCase, params}).then(result => { store.dispatch() }
```

File: src/components/demo/EventsButtons.js (hunk 3 above)
Comment (carlosvillu) [inline src/components/demo/EventsButtons.js]: really, I dont know.

**Pull request merged**